ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJpdGVtQWN0aW9uIjoiVmlldyBJdGVtIiwiY2F0YWxvZyI6eyJJdGVtQ2xhc3MiOnsiX2lkIjoib3ZlcnZpZXcifX0sInVzZXIiOnsiYXR0cmlidXRlcyI6eyJnYUNsaWVudElEIjoiMTc2OTg3MTYzOC4xNzQ1ODc0MDc4In0sImFub25JZCI6ImVlMWFlMjkxZDhhZjRlOGQifSwic291cmNlIjp7InVybCI6Imh0dHBzOi8vd3d3Lm1lcmNlZGVzLWJlbnouYXQvcGFzc2VuZ2VyY2Fycy9tb2RlbHMvc3V2L2dsYy9vdmVydmlldy5odG1sIiwidXJsUmVmZXJyZXIiOiIiLCJjaGFubmVsIjoiV2ViIiwiYmVhY29uVmVyc2lvbiI6MTYsImNvbmZpZ1ZlcnNpb24iOiI3MSIsImNvbnRlbnRab25lcyI6W119LCJjb25zZW50cyI6W10sImFjY291bnQiOnt9LCJfdG9vbHNFdmVudExpbmtJZCI6IjM1NDE2NTE1OTIwODc3NTgiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x56536d552cea <unknown>
#1 0x56536d0035f0 <unknown>
#2 0x56536cfe9e86 <unknown>
#3 0x56536cfe879a <unknown>
#4 0x56536cfe8eff <unknown>
#5 0x56536cfe8e54 <unknown>
#6 0x56536d0077ff <unknown>
#7 0x56536d0ac9aa <unknown>
#8 0x56536d07a662 <unknown>
#9 0x56536d0a0660 <unknown>
#10 0x56536d07a433 <unknown>
#11 0x56536d046ea3 <unknown>
#12 0x56536d047b01 <unknown>
#13 0x56536d517b3b <unknown>
#14 0x56536d51ba21 <unknown>
#15 0x56536d4fec32 <unknown>
#16 0x56536d51c594 <unknown>
#17 0x56536d4e2eef <unknown>
#18 0x56536d540d98 <unknown>
#19 0x56536d540f76 <unknown>
#20 0x56536d551b36 <unknown>
#21 0x7f95f9e9caa4 <unknown>
#22 0x7f95f9f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJWaXNpdGVkIC0gb3ZlcnZpZXciLCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjE3Njk4NzE2MzguMTc0NTg3NDA3OCJ9LCJhbm9uSWQiOiJlZTFhZTI5MWQ4YWY0ZThkIn0sInNvdXJjZSI6eyJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGMvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNzEiLCJjb250ZW50Wm9uZXMiOltdfSwiY29uc2VudHMiOltdLCJhY2NvdW50Ijp7fSwicGVyZm9ybWFuY2UiOnsic2RrUGFyc2VUaW1lIjoxfSwiX3Rvb2xzRXZlbnRMaW5rSWQiOiI5ODA5NzQyMjQwMTUzMDUiLCJkZWJ1ZyI6eyJleHBsYW5hdGlvbnMiOnRydWV9fQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x56536d552cea <unknown>
#1 0x56536d0035f0 <unknown>
#2 0x56536cfe9e86 <unknown>
#3 0x56536cfe879a <unknown>
#4 0x56536cfe8eff <unknown>
#5 0x56536cfe8e54 <unknown>
#6 0x56536d0077ff <unknown>
#7 0x56536d0ac9aa <unknown>
#8 0x56536d07a662 <unknown>
#9 0x56536d0a0660 <unknown>
#10 0x56536d07a433 <unknown>
#11 0x56536d046ea3 <unknown>
#12 0x56536d047b01 <unknown>
#13 0x56536d517b3b <unknown>
#14 0x56536d51ba21 <unknown>
#15 0x56536d4fec32 <unknown>
#16 0x56536d51c594 <unknown>
#17 0x56536d4e2eef <unknown>
#18 0x56536d540d98 <unknown>
#19 0x56536d540f76 <unknown>
#20 0x56536d551b36 <unknown>
#21 0x7f95f9e9caa4 <unknown>
#22 0x7f95f9f29c3c <unknown>

ERROR    root:XHRResponseCapturer.py:123 Error capturing response for https://daimleragemea.germany-2.evergage.com/api2/event/mb_austria_prod?event=eyJhY3Rpb24iOiJQcm9kdWN0IEluZm9ybWF0aW9uIFBhZ2UgLSBvdmVydmlldyAyNTQ2IiwiaXRlbUFjdGlvbiI6IlZpZXcgSXRlbSIsInNvdXJjZSI6eyJwYWdlVHlwZSI6InByb2R1Y3RfaW5mb3JtYXRpb24iLCJsb2NhbGUiOiJkZV9BVCIsImNvbnRlbnRab25lcyI6WyJnbG9iYWxfUElfTkwiLCJwaV9iYW5uZXJfTkwiLCJlcXMtc3BlY2lhbF9OTCJdLCJ1cmwiOiJodHRwczovL3d3dy5tZXJjZWRlcy1iZW56LmF0L3Bhc3NlbmdlcmNhcnMvbW9kZWxzL3N1di9nbGMvb3ZlcnZpZXcuaHRtbCIsInVybFJlZmVycmVyIjoiIiwiY2hhbm5lbCI6IldlYiIsImJlYWNvblZlcnNpb24iOjE2LCJjb25maWdWZXJzaW9uIjoiNzEifSwiZmxhZ3MiOnsicGFnZVZpZXciOnRydWV9LCJ1c2VyIjp7ImF0dHJpYnV0ZXMiOnsiZ2FDbGllbnRJRCI6IjE3Njk4NzE2MzguMTc0NTg3NDA3OCJ9LCJhbm9uSWQiOiJlZTFhZTI5MWQ4YWY0ZThkIn0sInBlcmZvcm1hbmNlIjp7InNka1BhcnNlVGltZSI6MX0sImRlYnVnIjp7ImV4cGxhbmF0aW9ucyI6dHJ1ZX0sImNhdGFsb2ciOnsiUHJvZHVjdCI6eyJfaWQiOiIyNTQ2IiwiZGltZW5zaW9ucyI6eyJCYXVtdXN0ZXJTaG9ydCI6WyIyNTQ2Il19fX0sImNvbnNlbnRzIjpbXSwiYWNjb3VudCI6e30sIl90b29sc0V2ZW50TGlua0lkIjoiMTI0MjA3NDQ2MTY1ODgxNTMifQ%3D%3D: Message: unknown error: unhandled inspector error: {"code":-32000,"message":"No resource with given identifier found"}
  (Session info: chrome=135.0.7049.95)
Stacktrace:
#0 0x56536d552cea <unknown>
#1 0x56536d0035f0 <unknown>
#2 0x56536cfe9e86 <unknown>
#3 0x56536cfe879a <unknown>
#4 0x56536cfe8eff <unknown>
#5 0x56536cfe8e54 <unknown>
#6 0x56536d0077ff <unknown>
#7 0x56536d0ac9aa <unknown>
#8 0x56536d07a662 <unknown>
#9 0x56536d0a0660 <unknown>
#10 0x56536d07a433 <unknown>
#11 0x56536d046ea3 <unknown>
#12 0x56536d047b01 <unknown>
#13 0x56536d517b3b <unknown>
#14 0x56536d51ba21 <unknown>
#15 0x56536d4fec32 <unknown>
#16 0x56536d51c594 <unknown>
#17 0x56536d4e2eef <unknown>
#18 0x56536d540d98 <unknown>
#19 0x56536d540f76 <unknown>
#20 0x56536d551b36 <unknown>
#21 0x7f95f9e9caa4 <unknown>
#22 0x7f95f9f29c3c <unknown>

ERROR    root:VerifyPersonalizationAndCapture.py:18 ‚ùå Error attaching screenshot to Allure: [Errno 2] No such file or directory: '/home/runner/work/QA-App-Allure/QA-App-Allure/Tests/BFV1_attempt_2.png'